### DISCUSSION TITLE
A connection with the server could not be established

### DISCUSSION THREAD

**Discussion by imuhammadarsalan** [microsoft/react-native-windows#10706] [Q&A] [answered]
Hello,

I am trying to send my project to my someone and i build the appx file but when it's loading i am getting this error mentioned in the title could anyone suggest any solution, please???
![temo](https://user-images.githubusercontent.com/13694232/194838121-e9e953a9-04f1-4b96-a8bf-51e1832ac331.png)

**Comment by jonthysell** [comment, marked as answer]
When launching a React Native app in Debug mode, you need the Metro Dev Server running to serve up your app's JS bundle (i.e. what happens if you launch with `run-windows`). If you want to send an app package to someone else, they either need to run Metro Dev Server themselves, or you should send a Release app package so the bundle is included.

**Reply by imuhammadarsalan** [reply]
How can I bundle the package ? like could you mention the command, please?
